test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Some tests would be good to add, but it doesn't have to be now. I'm guessing the rest of the team is mostly depending on the functionality being added.

Reply:
I'll create a task to perform tests. I'm just waiting for the rspec branch to get merged: #1